Assets/Scripts/WaveProfile.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
```

Comment:
Cool, so by my understanding the enemy spawns at a frequency of x or y seconds, so to work out number of enemies you take spawnFor and devide by spawnFrequancy